ThunderCatsTests/UIAlertControllerTests.m

```diff
@@ -0,0 +1,66 @@
+//
+//  UIAlertControllerTests.m
+//  ThunderCats
+//
+//  Created by Abbey Jackson on 2016-05-09.
+//  Copyright © 2016 Metova. All rights reserved.
+//
+
+#import <XCTest/XCTest.h>
+#import "UIAlertController+TCAdditions.h"
+
+@interface UIAlertControllerTests : XCTestCase
+
+@end
+
+@implementation UIAlertControllerTests
+
+- (void)testNotImplementedAlert
+{
+    UIAlertController *alert = [UIAlertController tc_notImplementedAlert];
+    
+    XCTAssertEqualObjects(alert.title, @"Not Implemented");
+    XCTAssertEqualObjects(alert.message, @"This feature is not yet implemented.");
+    XCTAssertEqualObjects(alert.actions[0].title, @"Ok");
+}
+
+- (void)testAlertWithTitle
```

Comment:
I know we don't have a particularly good way of testing the actions themselves, but we should write tests for both the case where we pass an action in and when we pass nil in.  We can see pretty easily by looking at the source that it shouldn't impact the things we are testing.  But that's all the more reason to add the extra case.  It doesn't make sense for `nil` or non-`nil` to effect the title, message, or button title... so let's write a test to assert that.